### PR TITLE
Feat/create profile info

### PR DESCRIPTION
**Pull request by Albertlarsen**
## Description
Created UserInfoBar component that can be used to display user info on all pages. I also fixed some other stuff eg. Updating flex values for different screen sizes. 

## Related Issue(s)
- #71 

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/components/reusables/UserInfoBar/UserInfoBar.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    <div className={classes.userInfoBarMargin}>
+      <div className={classes.userInfoBar}>
+        <div className={classes.altinnLogo}>
+          <AltinnLogo />
+        </div>
+        <div className={classes.userInfoContent}>
+          <div>
+            {userInfoName && <h5 className={classes.userText}>{userInfoName}</h5>}
+            {reporteeName && <h5 className={classes.userText}>for {reporteeName}</h5>}
+          </div>
```
Comment (allinox) [inline src/components/reusables/UserInfoBar/UserInfoBar.tsx:36]: Long company names pushes the users name far to the center of the screeen
image

Perhaps the text should be aligned right as in the studio-apps?
image

File: src/components/reusables/UserInfoBar/UserInfoBar.module.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  flex: 1;
+  display: flex;
+  justify-content: flex-end;
+  margin-right: 13px;
+}
+
+.userText{ 
+  font-weight: 500;
+  margin: 0;
+  margin-right: 50px;
```
Comment (allinox) [inline src/components/reusables/UserInfoBar/UserInfoBar.module.css:43]: 50px distance between the icon and the text feels a bit excessive.
How about 10px instead? (see picture)
image

File: public/Localizations/en.json [hunk 3]
```diff
@@ -65,12 +67,19 @@
     "additional_description": "Additional description",
     "data_retrieval_failed": "Data retrieval failed, try refreshing the page.",
     "loading": "Loading",
-    "error_message": "Error message status code",
+    "error_status_code": "Error message status code",
```
[the remark this answers is not in the document]
Comment (Albertlarsen) [reply public/Localizations/en.json:70]: good catch

File: src/components/apiDelegation/given/OverviewPage/OverviewPage.tsx [hunk 4]
```diff
@@ -3,22 +3,26 @@ import { useTranslation } from 'react-i18next';
...
-        <PageContent>
-          <OverviewPageContent layout={LayoutState.Given} />
-        </PageContent>
-      </Page>
-    </PageContainer>
+    <div>
+      <UserInfoBar></UserInfoBar>
+      <PageContainer>
+        <Page>
+          <PageHeader icon={<ApiIcon />}>{t('api_delegation.api_delegations')} test</PageHeader>
```
[the remark this answers is not in the document]
Comment (Albertlarsen) [reply src/components/apiDelegation/given/OverviewPage/OverviewPage.tsx:20]: Good catch

File: src/rtk/features/userInfo/userInfoSlice.ts [hunk 5]
```diff
@@ -0,0 +1,70 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+import { getCookie } from '@/resources/Cookie/CookieMethods';
+
+export interface InitialState {
+  userLoading: boolean;
+  reporteeLoading: boolean;
+  name: string;
```
[the remark this answers is not in the document]
Comment (Albertlarsen) [reply src/rtk/features/userInfo/userInfoSlice.ts:9]: Will be too similar to username but I can change it to personName

[the remark this answers is not in the document]
Comment (allinox) [reply src/rtk/features/userInfo/userInfoSlice.ts:9]: Sounds good! 👍

File: src/resources/NotFoundSite/NotFoundSite.module.css [hunk 6]
```diff
@@ -0,0 +1,47 @@
+.notFoundSiteWrapper {
+  margin-left: 20px;
```
Comment (allinox) [inline src/resources/NotFoundSite/NotFoundSite.module.css:2]: This is very much nitpick, but...
```suggestion
  margin-left: 40px;
```

File: src/resources/NotFoundSite/NotFoundSite.module.css [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+  margin-right: 20px;
```
Comment (allinox) [inline src/resources/NotFoundSite/NotFoundSite.module.css:3]:
```suggestion
  margin-right: 40px;
```

File: src/resources/NotFoundSite/NotFoundSite.module.css [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+.flexContainer {
+  display: flex;
+}
+
+.leftContainer {
+  flex: 3;
+}
+
+.rightContainer {
+  flex: 1;
```
Comment (allinox) [inline src/resources/NotFoundSite/NotFoundSite.module.css:24]: Another nitpick:
The page looks more symmetrical if this line of styling is skipped

**Review by allinox — APPROVED** [review]
A couple of nitpicks, but they're purely cosmetic.
Approved and well done! 😊

File: src/resources/NotFoundSite/NotFoundSite.module.css (hunk 8 above)
Comment (Albertlarsen) [reply src/resources/NotFoundSite/NotFoundSite.module.css:24]: I agree

**Pull request merged**